types/office-js-preview/index.d.ts

```diff
@@ -104143,6 +104143,14 @@ declare namespace Word {
          * @beta
          */
         readonly body: Word.Body;
+        /**
+         * Gets the top-level parent group shape of this child shape. It will be null if it isn't a child shape.
```

Comment:
What happens if this shape is a group within a group? The repetition of "child shape" makes me question whether this shape has to be an end node for this property to work.

Reply:
I've raised this with the feature crew. Waiting for a response.

Reply:
yes, it has to be an end node.

Reply:
@jipyua, Does that mean there is no way to navigate to and from a group within a group? Seems like an API gap.